Gems/Atom/Tools/ShaderManagementConsole/Code/Source/Document/ShaderManagementConsoleDocument.cpp

```diff
@@ -278,80 +278,7 @@ namespace ShaderManagementConsole
-        AZStd::set<AZ::RPI::ShaderVariantId> shaderVariantIds;
-        AZStd::vector<AZ::RPI::ShaderOptionGroup> shaderVariantListShaderOptionGroups;
-        for (const auto& materialAssetId : materialAssetIds)
-        {
-            const auto& materialInstanceShaderItems = GetMaterialInstanceShaderItems(materialAssetId);
-            for (const auto& shaderItem : materialInstanceShaderItems)
-            {
-                const auto& shaderAssetId = shaderItem.GetShaderAsset().GetId();
-                if (shaderAssetInfo.m_assetId == shaderAssetId)
-                {
-                    const auto& shaderVariantId = shaderItem.GetShaderVariantId();
-                    if (!shaderVariantId.IsEmpty() && shaderVariantIds.insert(shaderVariantId).second)
-                    {
-                        shaderVariantListShaderOptionGroups.push_back(shaderItem.GetShaderOptionGroup());
-                    }
-                }
-            }
-        }
-
-        // Generate the shader variant list data by collecting shader option name-value pairs.s
-        AZ::RPI::ShaderVariantListSourceData shaderVariantListSourceData;
-        shaderVariantListSourceData.m_shaderFilePath = shaderAssetInfo.m_relativePath;
-        int stableId = 1;
-        for (const auto& shaderOptionGroup : shaderVariantListShaderOptionGroups)
-        {
-            AZ::RPI::ShaderVariantListSourceData::VariantInfo variantInfo;
-            variantInfo.m_stableId = stableId;
-
-            const auto& shaderOptionDescriptors = shaderOptionGroup.GetShaderOptionDescriptors();
-            for (const auto& shaderOptionDescriptor : shaderOptionDescriptors)
-            {
-                const auto& optionName = shaderOptionDescriptor.GetName();
-                const auto& optionValue = shaderOptionGroup.GetValue(optionName);
-                if (!optionValue.IsValid())
-                {
-                    continue;
-                }
-
-                const auto& valueName = shaderOptionDescriptor.GetValueName(optionValue);
-                variantInfo.m_options[optionName.GetStringView()] = valueName.GetStringView();
-            }
-
-            if (!variantInfo.m_options.empty())
-            {
-                shaderVariantListSourceData.m_shaderVariants.push_back(variantInfo);
-                stableId++;
-            }
-        }
-
-        // Even though the data originated from a different file source it has now been transformed into a shader variant list so update the
-        // extension to match. This will allow the document to be resaved immediately without doing a save as child or derived type when
-        // loaded from a shader source file.
-        AzFramework::StringFunc::Path::ReplaceExtension(m_absolutePath, AZ::RPI::ShaderVariantListSourceData::Extension);
-
-        SetShaderVariantListSourceData(shaderVariantListSourceData);
+        
         m_modified = {};
 
         return IsOpen() ? OpenSucceeded() : OpenFailed();
```

Comment:
This should return OpenFailed() and the shader extension support should be removed from the build document type function at the top of the file.